cluster/ci/config/mirroring/origin_4_2

```diff
@@ -141,6 +141,7 @@ registry.svc.ci.openshift.org/origin/4.2:recycler quay.io/openshift/origin-recyc
 registry.svc.ci.openshift.org/origin/4.2:service-ca-operator quay.io/openshift/origin-service-ca-operator:4.2 quay.io/openshift/origin-service-ca-operator:4.2.0 quay.io/openshift/origin-service-ca-operator:latest
 registry.svc.ci.openshift.org/origin/4.2:service-catalog quay.io/openshift/origin-service-catalog:4.2 quay.io/openshift/origin-service-catalog:4.2.0 quay.io/openshift/origin-service-catalog:latest
 registry.svc.ci.openshift.org/origin/4.2:setup-etcd-environment quay.io/openshift/origin-setup-etcd-environment:4.2 quay.io/openshift/origin-setup-etcd-environment:4.2.0 quay.io/openshift/origin-setup-etcd-environment:latest
+registry.svc.ci.openshift.org/origin/4.2:sig-storage-local-static-provisioner quay.io/openshift/origin-sig-storage-local-static-provisioner:4.2 quay.io/openshift/origin-sig-storage-local-static-provisioner:4.2.0 quay.io/openshift/origin-sig-storage-local-static-provisioner:latest
```

Comment:
This naming doesn't look correct.  Is this something that will be in the release payload?  IF so it needs to follow the naming conventions for other provisioners.

Reply:
What do you mean it does not follow existing "provisioners" naming convention? You mean it has `sig-storage` word in it? Why is that a problem? 

I think tooling on `openshift/release` repo has some restriction on what released image can be called if git repo is named `sig-storage-local-static-provisioner `. I remember fighting it while integrating this repo in ci-operator.